packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -95,8 +112,8 @@ class BlockListBlock extends Component {
 
 	render() {
 		const {
-			attributes,
-			blockType,
+			attributes = {},
+			blockType = {},
```

Comment:
Is there a possibility to not have the `attributes` and `blockType`? If yes - should we render block with empty object as `attributes` and `blockType` or we can just return `null`?

Reply:
Yup, will do that.